source/docs/software/advanced-controls/controllers/feedforward.rst

```diff
@@ -11,6 +11,8 @@ There are two types of feedforwards: model-based feedforward and feedforward for
 
 .. note:: The WPILib feedforward classes closely match the available mechanism characterization tools available in the :ref:`frc-characterization toolsuite <docs/software/pathplanning/robot-characterization/introduction:Introduction to Robot Characterization>` - the characterization toolsuite can be used to quickly and effectively determine the correct gains for each type of feedforward.  The toolsuite will indicate the appropriate units for each of the gains.
 
+.. note:: If you are unable to empirically characterize your mechanism (due to space and/or time constraints), reasonable estimates of ``kG``, ``kV``, and ``kA`` can be obtained by fairly simple computation, and are also available from `ReCalc<https://www.reca.lc>`__.
```

Comment:
Is there a reason kS isn't mentioned? I can see how kG and kV can be simply computed, but kA doesn't seem so simple. In the spirit of learning, it would be better to describe the method.

Reply:
kS is impossible to model.  kA is just as easy as kV and kG.  ReCalc is open-source; the method is just some multiplications.